packages/buidler-core/test/internal/core/typescript-support.ts

```diff
@@ -59,8 +60,8 @@ describe("Typescript support", function() {
       });
 
       assert.deepEqual(tests.sort(), [
-        await fsExtra.realpath("test/js-test.js"),
-        await fsExtra.realpath("test/ts-test.ts")
+        await fsExtra.realpath("test/js-test.js").then(slash),
```

Comment:
I don't get why these need to be `slash`ed. These are file paths.

Reply:
`TASK_TEST_GET_TEST_FILES` returns the files with unix slashes, even in windows. I thought it was easier to change the tests than the functionality.

Reply:
That's surprising, and will probably lead to bugs in the future.

I think the problem comes from our `glob` module. I'd rather normalize the paths to use the system's separators before returning them from it. This means modifying `glob` and `globSync`.

Reply:
Yeah, I thought about that but it seemed a little out of scope. I will do it now :+1:

Reply:
I fixed it adding the `realpath` option to the glob utils. This fixes the problem and the test passes in both linux and windows. The docs have a caveat:

> realpath Set to true to call fs.realpath on all of the results. In the case of a symlink that cannot be resolved, the full absolute path to the matched entry is returned (though it will usually be a broken symlink)

But that doesn't seem like it will be an issue here.